GOVERNANCE.md

```diff
@@ -279,7 +279,6 @@ Staff membership does not grant any additional abilities when it comes to voting
 
 - `@alumni` role on [Discord](https://astro.build/chat)
 - New name color on Discord: **light blue**.
-- Invitation to the private `#alumni` channel on Discord.
```

Comment:
keep this? Being alumni = being a part of this channel, unless you're suggesting killing that channel.

Reply:
I thought it never existed. I will revert the change